### ISSUE TITLE
[FEATURE]: Implement smooth D and ID illuminants in CIE 204:2013 "Methods for Re-defining CIE D Illuminants"

### ISSUE THREAD

**Issue by Artoria2e5** [colour-science/colour#1242]
### Description

CIE 204:2013 "Methods for Re-defining CIE D Illuminants" (https://cie.co.at/publications/methods-re-defining-cie-d-illuminants) defines a number of "highly smoothed" versions of the D and ID illuminants. They are referenced in https://www.oscars.org/sites/oscars/files/ssi_overview_2020-09-16.pdf.

`colour` also doesn't seem to have a thing for "D illuminant of arbitrary CCT"? There are basis functions, but I don't see anything just 

(No, I don't need it for anything. I'm just being a squirrel and collecting stuff.)

**Comment by KelSolaar** [comment]
Hello @Artoria2e5,

Thanks, I vaguely remember seeing those versions of the CIE illuminant D series but we never implemented them. That could be an interesting thing to add for sure. We are currently linearly interpolating them. Any thoughts @tjdcs?

We do have generation of CIE illuminant D series  for arbitrary CCT, see the docstring here: https://colour.readthedocs.io/en/develop/generated/colour.sd_CIE_illuminant_D_series.html#colour.sd_CIE_illuminant_D_series

> is not getting rendered on the website

I know, it sadly happened when we switched from a fully automated API documentation generation to a more manual one.

Cheers,

Thomas

**Comment by tjdcs** [comment]
I can take this issue at some point in the near future. It's relevant to my other work. @KelSolaar Can we please purchase the relevant standards? Or if we would prefer, I can access them via my other legal means. I just wouldn't be able to share them with the colour-maintenence team.

**Comment by KelSolaar** [comment]
Yes I have planned to purchase it! Will loop you in when it is done!
